<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgFrm</class>
 <widget class="QDialog" name="DlgFrm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>380</width>
    <height>480</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>11</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>系统配置</string>
      </attribute>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>351</width>
         <height>96</height>
        </rect>
       </property>
       <property name="title">
        <string>悬浮窗相关</string>
       </property>
       <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0">
        <item row="0" column="0">
         <widget class="QCheckBox" name="chkSuspenShow">
          <property name="text">
           <string>悬浮窗显示</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="cboxSuspenPaint">
          <property name="minimumSize">
           <size>
            <width>85</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <item>
           <property name="text">
            <string>锯齿</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>抗锯齿</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>动画</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QCheckBox" name="chkGoodGif">
          <property name="text">
           <string>打开彩蛋</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QCheckBox" name="chkClock">
          <property name="text">
           <string>打开时钟</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="chkSuspenExit">
          <property name="text">
           <string>退出到托盘</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>380</y>
         <width>121</width>
         <height>26</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 0, 0);
font: 11pt &quot;微软雅黑&quot;;</string>
       </property>
       <property name="text">
        <string>程序重启后生效!</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
